### DISCUSSION TITLE
How to set a PDF image file in the tex directory instead of the doc directory in l3build?

### DISCUSSION THREAD

**Discussion by note286** [latex3/l3build#231] [Q&A] [answered]
I have a `xxxlogo.pdf` file, which is a logo image file for users to use with `.cls`. By default, l3build will consider this as a document and place it in the `doc` directory. For this reason, I set some options.

```lua
module       = "xxxx"
ctanzip      = "xxxx"
tdsroot      = "xelatex"
typesetexe   = "xelatex"
sourcefiles  = {"*.dtx", "*.ins", "xxxlogo.pdf"}
installfiles = {"*.sty", "*.cls", "xxxlogo.pdf"}
```

At present, the `xxxlogo.pdf` file can appear in the `tex` directory, but it also exists in the `doc` directory. How can I make the `xxxlogo.pdf` file only appear in the `tex` directory, but not in the `doc` directory?

**Comment by josephwright** [comment, marked as answer]
Probably easiest is
```latex
tdslocations = {"tex/latex/xxxx/xxxlogo.pdf"}
```
as this simply forces the location for the file.

**Reply by note286** [reply]
Thank you very much. Your answer has solved the problem that has been bothering me for a long time.
